docs/installation/windows.md

```diff
@@ -30,6 +30,48 @@ Be noted that when using the ZIP archive, you won't get the prerequisites check
 So in order for remoting over WinRM to work properly on Windows versions prior to Windows 10,
 you need to make sure the [prerequisites](#prerequisites) are met.
 
+## Deploying on Windows IoT
+
+Windows IoT already comes with Windows PowerShell which we will use to deploy PowerShell Core 6.
+
+* Create PSSession to target device
```

Comment:
Looks like Travis CI detected spelling error in this line about "PSSession".

Reply:
Will fix

Reply:
PSSession is a class name, so maybe we should put it in a code block (enclose it with ``). If so, we don't need to change the `.spelling` file.

Reply:
Good suggestion, will change.